### PR TITLE
Added EDXAPP_LMS_ROOT_URL variable

### PR DESCRIPTION
**Pull request by clintonb**
The EDXAPP_LMS_ROOT_URL has been added to simplify the addition of LMS-hosted URLs to IDA configurations.

ECOM-3602

### CODE REVIEW

**Comment by clintonb** [comment]
@fredsmith @jibsheet please review
FYI @bderusha @peter-fogg

**Comment by clintonb** [comment]
`EDXAPP_LMS_ROOT_URL` needs to be set in the private repo.

File: playbooks/roles/edxapp/defaults/main.yml [hunk 1]
```diff
@@ -247,7 +247,8 @@ EDXAPP_BULK_EMAIL_DEFAULT_FROM_EMAIL: 'no-reply@example.com'
 EDXAPP_BULK_EMAIL_LOG_SENT_EMAILS: false
 EDXAPP_UNIVERSITY_EMAIL: 'university@example.com'
 EDXAPP_PRESS_EMAIL: 'press@example.com'
-EDXAPP_LMS_ISSUER: "{{ EDXAPP_LMS_BASE_SCHEME | default('https') }}://{{ EDXAPP_LMS_BASE }}/oauth2"
+EDXAPP_LMS_ROOT_URL: "{{ EDXAPP_LMS_BASE_SCHEME | default('https') }}://{{ EDXAPP_LMS_BASE }}"
```
Comment (feanil) [inline playbooks/roles/edxapp/defaults/main.yml:251]: What about the port number?  You could be running LMS on a non-standard port, is the expectation that in that case you'd override the whole URL?

Comment (clintonb) [inline playbooks/roles/edxapp/defaults/main.yml:251]: `EDXAPP_LMS_BASE` includes the port: https://github.com/edx/configuration/blob/master/playbooks/roles/edxapp/defaults/main.yml#L26

**Comment by feanil** [comment]
:+1:

**Pull request merged**